File: URI_1045_triangle_types.js
const lines = ["7.0 5.0 7.0"];

const inputArr = lines[0].split(" ");
let a = Number(inputArr[0]);
let b = Number(inputArr[1]);
let c = Number(inputArr[2]);

const arrToSort = [ a, b, c ];
arrToSort.sort((a, b) => b - a);
a = arrToSort[0];
b = arrToSort[1];
c = arrToSort[2];

if (a >= b + c) {
    console.log("NAO FORMA TRIANGULO");
    return;
}

const aSqr = a * a, bSqrPlusCSqr = b * b + c * c;
if (aSqr === bSqrPlusCSqr) {
    console.log("TRIANGULO RETANGULO");
} else if (aSqr > bSqrPlusCSqr) {
    console.log("TRIANGULO OBTUSANGULO");
} else if (aSqr < bSqrPlusCSqr) {
    console.log("TRIANGULO ACUTANGULO");
}

if (a === b && b === c) {
    console.log("TRIANGULO EQUILATERO");
} else if ((a === b && a !== c) || (b === c && b !== a) || (c === a && c !== b)) {
    console.log("TRIANGULO ISOSCELES");
}

File: URI_1045_triangle_types.java
import java.util.Arrays;
import java.util.Scanner;

public class URI_1045_triangle_types {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		double a = scanner.nextDouble();
		double b = scanner.nextDouble();
		double c = scanner.nextDouble();

		Double[] arrayToSort = { a, b, c };
		Arrays.sort(arrayToSort, (o1, o2) -> (int) (o2 - o1));
		a = arrayToSort[0];
		b = arrayToSort[1];
		c = arrayToSort[2];

		if (a >= b + c) {
			System.out.println("NAO FORMA TRIANGULO");
			return;
		}

		double aSqr = a * a, bSqrPlusCSqr = b * b + c * c;
		if (aSqr == bSqrPlusCSqr) {
			System.out.println("TRIANGULO RETANGULO");
		} else if (aSqr > bSqrPlusCSqr) {
			System.out.println("TRIANGULO OBTUSANGULO");
		} else if (aSqr < bSqrPlusCSqr) {
			System.out.println("TRIANGULO ACUTANGULO");
		}

		if (a == b && b == c) {
			System.out.println("TRIANGULO EQUILATERO");
		} else if ((a == b && a != c) || (b == c && b != a) || (c == a && c != b)) {
			System.out.println("TRIANGULO ISOSCELES");
		}
	}
}